sw/device/tests/BUILD

```diff
@@ -151,6 +151,64 @@ opentitan_test(
     ],
 )
 
+opentitan_test(
+    name = "aes_force_prng_reseed_test",
+    srcs = ["aes_force_prng_reseed_test.c"],
+    exec_env = dicts.add(
```

Comment:
I think that you want to add the execution environments for sival:
https://github.com/lowRISC/opentitan/blob/c52ee256f90c67f05646e2aa3e8b0d2c51aef1a1/sw/device/tests/BUILD#L93-L100

Reply:
Sure, noted